drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1282,6 +1282,8 @@ bool RasterizerSceneGLES3::_setup_material(RasterizerStorageGLES3::Material *p_m
 				case ShaderLanguage::TYPE_SAMPLER2DARRAY: {
 					// TODO
 				} break;
+
+				default: {}
```

Comment:
```
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_BOOL' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_BVEC2' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_BVEC3' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_BVEC4' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_FLOAT' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_INT' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_IVEC2' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_IVEC3' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_IVEC4' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_MAT2' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_MAT3' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_MAT4' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_UINT' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_UVEC2' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_UVEC3' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_UVEC4' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_VEC2' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_VEC3' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_VEC4' not handled in switch [-Wswitch]
drivers/gles3/rasterizer_scene_gles3.cpp:1235:11: warning: enumeration value 'TYPE_VOID' not handled in switch [-Wswitch]
```
@reduz

Reply:
Nevermind the ping here, it's clearly correct. This switch only cares about texture types.